app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -389,27 +389,27 @@ export class CloneRepository extends React.Component<
     if (tab === CloneRepositoryTab.DotCom) {
       this.setState(
         prevState => ({
-          dotComTabState: merge<IGitHubTabState, K>(
-            prevState.dotComTabState,
-            state
-          ),
+          dotComTabState: {
+            ...prevState.dotComTabState,
+            ...state,
```

Comment:
These changes were necessary because the type checker got more strict with what was permissible here, and ` state: Pick<IBaseTabState, K>` no longer satisfies the type that `merge` expects (i.e. both parameters should satisfy `IGitHubTabState`):

```
Argument of type 'Pick<IBaseTabState, K>' is not assignable to parameter of type 'Pick<IGitHubTabState, K>'.
  Type 'IBaseTabState' is missing the following properties from type 'IGitHubTabState': kind, filterText, selectedItem
```